app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -722,4 +722,8 @@ public void setFileBrowserLastBrowsedFolder(File f) {
     public File getFileBrowserLastBrowsedFolder() {
         return new File(getString(R.string.pref_key__file_browser_last_browsed_folder, getNotebookDirectoryAsStr()));
     }
+
+    public int getDateDialogRecentCount() {
+        return 5;
```

Comment:
@gsantner Do you want to make this user-configurable? I didn't think so...

Reply:
No, just have the 5 where it's required / date, no appsetting method required